packages/next-urql/src/useUrqlValue.ts

```diff
@@ -0,0 +1,47 @@
+'use client';
+
+import React from 'react';
+import { useDataHydrationContext } from './DataHydrationContext';
+import { SSRContext } from './Provider';
+
+export const symbolString = 'urql_transport';
+export const urqlTransportSymbol = Symbol.for(symbolString);
+
+export type UrqlResult = { data: any; error: any };
+
+export function useUrqlValue(operationKey: number, value?: UrqlResult): void {
+  const ssrExchange = React.useContext(SSRContext);
+  const rehydrationContext = useDataHydrationContext();
+
+  if (!ssrExchange) {
+    throw new Error('forgot to pass an "ssrExchange" to the UrqlProvider.');
```

Comment:
This can also be caused by UrqlProvider not being used/not being a parent component, mentioning that in the error would have saved me an embarassing amount of time